notes/force-move-implementation.md

```diff
@@ -164,9 +175,110 @@ Checking signatures is likely to be a signification part of the logic. Here's a
 
 **Can we optimize this further by changing the way states / sigs are passed in?** For example passing the arrays in last-state-first might help.
 
-## Updates call signatures
+## ForceMove interface
+
+With these considerations in mind, the ForceMove implementation should be something like the below.
+
+### Types and storage
+
+```javascript
+
+struct FixedPart {
+ string chainId;
+ address[] participants;
+ uint256 channelNonce;
+}
+
+struct VariablePart {
+  uint256 turnNum;
+  bool isFinal;
```

Comment:
This should be the `outcome` and the `appData` (and not the `turnNum` and `isFinal`).

Reply:
Yes sorry this wasn't intended, have fixed in 4de94bf